core/authenticate/strategy/passkey.go

```diff
@@ -0,0 +1,54 @@
+package strategy
+
+import (
+	"strings"
+	"time"
+
+	"github.com/go-webauthn/webauthn/webauthn"
+)
+
+const (
+	PasskeyAuthMethod string = "passkey"
+)
+
+type UserData struct {
+	Id          string
+	Name        string
+	DisplayName string
+	Credentials []webauthn.Credential
+	CreatedAt   time.Time
```

Comment:
Unused?

Reply:
removed CreatedAt since we will use user.user createdAt time in our databse, no need of it here